configs/platforms/solaris-11-i386.rb

```diff
@@ -1,12 +1,13 @@
 platform "solaris-11-i386" do |plat|
   plat.inherit_from_default
+  plat.make "gmake"
 
-  packages = %w(
-    pl-binutils-i386
-    pl-cmake
-    pl-gcc-i386
-    pl-pkg-config
-  )
+   packages = %w(
+     pl-binutils-i386
+     pl-cmake
+     pl-gcc-i386
+     pl-pkg-config
+   )
```

Comment:
nit extra white space makes it look like there are changes, but only `gmake` is new